main/src/ca/uwaterloo/flix/language/phase/jvm/BackendObjType.scala

```diff
@@ -654,6 +655,8 @@ object BackendObjType {
     }
   }
 
+  case class Defn(sym: Symbol.DefnSym) extends BackendObjType
```

Comment:
Why are we adding such empty classes-- should we not be moving in the opposite direction?

Reply:
Like this also costs object allocation